api/checks/webhook.go

```diff
@@ -338,6 +379,13 @@ func createCheckRun(ctx context.Context, suite shared.CheckSuite, opts github.Cr
 	return true, nil
 }
 
+// isUserWhitelisted checks if a commit from a given GitHub username should
+// result in wpt.fyi or staging.wpt.fyi summary results showing up. Currently
+// this is enabled for all users on prod, but only for some users on staging to
+// avoid having a confusing double-set of checks appear on the GitHub UI.
+//
+// TODO: Should we remove checksForAllUsersFeature and just differentiate based
+// on whether the AppId is prod/staging?
```

Comment:
Both apps are enabled, so that would show double checks for everyone. Just disabling the staging checks would do it though.